.github/workflows/bcnotify.yaml

```diff
@@ -1,13 +1,13 @@
 name: "bc-notification"
-on: 
+on:
   issues:
     types: [edited, labeled]
 
 jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
@timheuer I'm not actually sure the checkout is required for this one, since it doesn't seem to interact with the code

Reply:
It's possible you are right, however I could never get things to trigger unless that context were there.  Keep it in for now?

Reply:
Not sure since I'm also new to this. Don't think it's a big deal to keep it, just thought it might be "slowing it down" because it clones the repo for each issue change. That's a GitHub problem though :)